Automattic/woocommerce-payments
# Show Currency filter in Disputes page

Fixes #3697 

#### Changes proposed in this Pull Request





Make it possible to filter by currency in the Disputes list. All the work was already there, it only needed few small fixes to turn it on.





Before:
![Screen Shot 2022-01-31 at 14 40 55](https://user-images.githubusercontent.com/12542046/151821152-8e6ab39a-49cd-487b-a89c-c1155dd99b4f.png)

After:
![Screen Shot 2022-01-31 at 14 35 46](https://user-images.githubusercontent.com/12542046/151821162-4ed6864a-5cc8-45e6-b5b6-c70f8c240924.png)

#### Testing instructions





* Navigate to Disputes list
* Make sure there are different disputes with different currencies
* Double check the currencies displayed on the filter are for those in the disputes only (ie, if disputes are in USD/JYP, filter should only display those currencies and not EUR for example)
* Filter by a currency
* Double-check disputes list contains disputes in the selected currency
* Refresh your page and double-check the filter is still there (it's using a query param, so it should be there)
* Double-check filter works with advanced filters (like status, and date)

-------------------


**Post merge**



- [x] Link to testing instructions from [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) : [Testing currency filter in disputes page](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions-for-WC-Payments-3.7.0/a3ebbb17ebc8dca11df23bfbac145b7bb096d3ac)
